packages/mgt-components/src/components/mgt-tasks/mgt-tasks.ts

```diff
@@ -1313,6 +1308,7 @@ export class MgtTasks extends MgtTemplatedComponent {
     const picker = mgtHtml`
       <mgt-people-picker
         class="people-picker picker-${taskId}"
+        .groupId=${ifDefined(this.groupId)}
```

Comment:
I think that we need to be deriving the groupId from the targetId when it is provided